clear/main.py

```diff
@@ -550,6 +552,27 @@ def route(prefix, vrf, namespace):
 helper = util_base.UtilHelper()
 helper.load_and_register_plugins(plugins, cli)
 
+# ("sonic-clear asic-sdk-health-event")
+@cli.command()
+@click.option('--namespace', '-n', 'namespace', required=False, default=None, show_default=False,
+              help='Option needed for multi-asic only: provide namespace name',
+              type=click.Choice(multi_asic_util.multi_asic_ns_choices()))
+@clicommon.pass_db
+def asic_sdk_health_event(db, namespace):
+    if multi_asic.get_num_asics() > 1:
+        namespace_list = multi_asic.get_namespaces_from_linux()
+    else:
+        namespace_list = [DEFAULT_NAMESPACE]
+
+    for ns in namespace_list:
+        if namespace and namespace != ns:
+            continue
+
+    state_db = db.db_clients[ns]
```

Comment:
indent issue? should indent with 8 white spaces?

Reply:
nice catch.
fixed